pep-0701.rst

```diff
@@ -225,9 +225,12 @@ for details on the syntax):
 The new tokens (``FSTRING_START``, ``FSTRING_MIDDLE``, ``FSTRING_END``) are defined
 :ref:`later in this document <701-new-tokens>`.
 
-This PEP leaves up to the implementation the level of f-string nesting allowed.
-This means that limiting nesting is **not part of the language specification**
-but also the language specification **doesn't mandate arbitrary nesting**. 
+This PEP leaves up to the implementation the level of f-string nesting allowed but
+**specifies a lower bound of 5 levels of nesting**. This is to ensure that users cannot
```

Comment:
```suggestion
**specifies a lower bound of 5 levels of nesting**. This is to ensure that users can
```

Seems like a typo, as ISTM that you _do_ want to ensure that users have a reasonable expectation of being able to nest f-strings to a "resonable" depth, and that specifying a required lower bound would rather than would not ensurethat.